tests/end_to_end/end_to_end.cpp

```diff
@@ -1139,6 +1158,7 @@ DECLARE_CGROUP_SOCK_ADDR_LOAD_TEST(
 DECLARE_CGROUP_SOCK_ADDR_LOAD_TEST(
     SAMPLE_PATH "cgroup_sock_addr", "authorize_recv_accept6", EBPF_ATTACH_TYPE_CGROUP_INET6_RECV_ACCEPT);
 
+#if !defined(CONFIG_BPF_JIT_DISABLED)
```

Comment:
nit: instead of removing these tests from native build, we could possibly use `native_module_helper_t` helper to get an appropriate file name (.sys or .o) based on whether JIT is enabled or disabled.

Reply:
Yes that's what #1121 already covers.